.gitignore

```diff
@@ -144,3 +144,6 @@ dist
 # Code interpreter data
 /examples/tmp/code_interpreter/*
 /examples/tmp/local/*
+
+# Observe data
+trace.json
```

Comment:
Use `/examples/tmp/observability/trace.json` to save the trace.

And add this line to the `.gitignore`

```
/examples/tmp/observability/*
```